src/lxc/console.c

```diff
@@ -583,7 +746,7 @@ void lxc_console_delete(struct lxc_console *console)
 
 	ret = lxc_console_write_ringbuffer(console);
```

Comment:
Do we need this?  lxc_console_cb_con write console log into ringbuffer and log file, i think log file have the same logs with ringbuffer.
```
lxc_console_cb_con() {
     if (console->buffer_size > 0)
 			w_rbuf = lxc_ringbuf_write(&console->ringbuf, buf, r);
     if (console->log_fd > 0)
			w_log = lxc_console_write_log_file(console, buf, r);
}
```

Reply:
> Do we need this? lxc_console_cb_con write console log into ringbuffer and log file, i think log file have the same logs with ringbuffer.

If `console->log_fd > 0` then the user wants us to write to the logfile. If they only care about the ringbuffer they can just choose to not use a console logfile.

Reply:
Oh, sorry I misread your comment. The idea is that if the container e.g. aborts you get a dump of the last ringbuffer contents.